JHUGenLexicon/interface/JHUGenLexiconCouplings.h

```diff
@@ -10,7 +10,8 @@
 #define DEFVAL_LAMBDA_VI 10000.
 #define DEFVAL_ALPHA .0072973525693 //Used to set e^2
 #define DEFVAL_VEV_LAM .0024622 //Vacuum Expectation Value GeV / Lambda Warsaw Basis
-
+#define DEFVAL_DELTA_V 0 // Delta V used in Warsaw Basis
+#define DEFVAL_DELTA_M 0 // Delta V used in Warsaw Basis
```

Comment:
I suppose this is dM? You write Delta V in the comments.

Reply:
Yes I will change this comment